molgenis-api-data/src/main/java/org/molgenis/api/data/v1/AttributeResponse.java

```diff
@@ -274,6 +265,7 @@ public Boolean getLookupAttribute() {
     return lookupAttribute;
   }
 
+  @SuppressWarnings("unused")
   public Boolean isAggregatable() {
```

Comment:
IntelliJ reports that the following methods are unused as well:
```java
public List<String> getEnumOptions()
public String getMappedBy()
public Object getDefaultValue()
public Range getRange()
public String getExpression()
public String getNullableExpression()
public String getVisibleExpression()
public String getValidationExpression()
```
Manually inspecting the usages of other getters results in none of them being used. Is there a reason for only suppressing this method?

Considering that this class is in practice only used the GSON message converter for serialization we could drop all the accessors according to the [GSON user guide](https://github.com/google/gson/blob/master/UserGuide.md#finer-points-with-objects).

What do you think?